http/handler.go

```diff
@@ -181,7 +182,7 @@ func wrapGenericHandler(core *vault.Core, h http.Handler, maxRequestSize int64,
 			}
 			r = newR
 
-		case strings.HasPrefix(r.URL.Path, "/ui/"), r.URL.Path == "/":
+		case strings.HasPrefix(r.URL.Path, "/ui"), r.URL.Path == "/":
```

Comment:
Did you actually want this? handlUIRedirect ends up with `/ui/` right?

Reply:
Hmm, maybe I should add `r.URL.Path ==  "/ui"` to the case instead? We want to redirect `/ui` as well as `/` here and w/o hitting this case statement the redirect doesn't happen.

Reply:
Ah, this handler is coming before handleUIRedirect. Never mind me, carry on :-)